src/main/java/seedu/duke/command/add/AddModuleCommand.java

```diff
@@ -10,6 +10,10 @@
 
 public class AddModuleCommand extends AddCommand {
     private String module;
+    public static final String FORMAT = COMMAND_WORD + " -m <module code>";
+    public static final String HELP =   "Add a module to the scheduler." +
```

Comment:
Should also add context that only mods that can be found on NUSMods can be added here